core/store/migrate/migrations/0100_bootstrap_config.sql

```diff
@@ -0,0 +1,117 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE bootstrap_contract_configs
+(
+    bootstrap_spec_id       INTEGER PRIMARY KEY,
+    config_digest           bytea                    NOT NULL,
+    config_count            bigint                   NOT NULL,
+    signers                 bytea[],
+    transmitters            text[],
+    f                       smallint                 NOT NULL,
+    onchain_config          bytea,
+    offchain_config_version bigint                   NOT NULL,
+    offchain_config         bytea,
+    created_at              timestamp with time zone NOT NULL,
+    updated_at              timestamp with time zone NOT NULL,
+    CONSTRAINT bootstrap_contract_configs_config_digest_check CHECK ((octet_length(config_digest) = 32))
+);
+
+ALTER TABLE ONLY bootstrap_contract_configs
+    ADD CONSTRAINT bootstrap_contract_configs_oracle_spec_fkey
+        FOREIGN KEY (bootstrap_spec_id)
+            REFERENCES bootstrap_specs (id)
+            ON DELETE CASCADE;
+
+-- add missing unique constraint to bootstrap specs
+CREATE UNIQUE INDEX idx_jobs_unique_bootstrap_spec_id ON jobs USING btree (bootstrap_spec_id);
+
+-- migrate existing OCR2 bootstrap jobs to the new bootstrap spec
+-- create helper column
+ALTER TABLE bootstrap_specs
+    ADD COLUMN job_id INTEGER;
```

Comment:
I don't think this is necessary, you can use a join in an update clause in postgres

Reply:
yeah this seems fine as a helper column. Whole thing is running inside a tx so we're safe

Reply:
SGTM